src/matching_market.sol

```diff
@@ -17,8 +17,10 @@
 
 pragma solidity ^0.5.12;
 
-import "./expiring_market.sol";
 import "ds-note/note.sol";
+import "ds-auth/auth.sol";
+import "./simple_market.sol";
+import "./uniswap/UniswapSimplePriceOracle.sol";
```

Comment:
I'd use a simple interface in this contract instead of importing the real file. The end bytecode will be the same but if you use these files for verifying it will end up with a lot of unnecessary code.

Reply:
Yes, totally makes sense. I get down to it now.